README.md

```diff
@@ -0,0 +1,66 @@
+# ✨ FUN Config ✨
+
+This repository is the place where we store various configuration files shared
+among the FUN development team.
+
+## GitHub Labels
+
+Inspired by TailorDev's work on [communication with GitHub
+issues](https://tailordev.fr/blog/2016/09/27/communication-with-github-issues-1/),
+we decided to define our own set of labels that will be used in all our GitHub
+repositories. In this section, we will document how to install and use the
+[`labels`](https://github.com/popomore/github-labels) tool to automate labels
+creation for a GitHub repository.
+
+### Getting started with `labels`
+
+#### GitHub personal access token
+
+To interact with GitHub API, you will need to [generate a personal access
+token](https://github.com/settings/tokens) (make sure it has the `repo`
+permission).
+
+Once generated, we will store this token in an environment variable that will be
+defined in your favorite shell rc file:
+
+```bash
+# .bashrc or .zshrc
+export GITHUB_LABELS_TOKEN="thisismytoken"
+```
+
+Load a new shell or source your rc file to take into account this new
+environment variable (_e.g._ `source $HOME/.zshrc`).
+
+#### Install the `labels` tool
+
+Make sure [Yarn](https://yarnpkg.com) is installed on your machine, then, at the
+root of this repository, install `labels`:
+
+```bash
+$ yarn install
```

Comment:
Wasn't clear to me at first that this works because I'm at the root of the github repo of fun-config.
My first comment was: 
should be "yarn install github-labels"

Reply:
Ok, I will clarify this, but the dependencies already listed in the `package.json` (+ `yarn.lock`) file, hence only `yarn install` is required for a local installation (which i recommend).